python/cudf/cudf/core/indexing.py

```diff
@@ -113,7 +116,19 @@ class _SeriesLocIndexer(object):
     def __init__(self, sr):
         self._sr = sr
 
-    def __getitem__(self, arg):
+    def __getitem__(self, arg: Any) -> Union[ScalarLike, DataFrameOrSeries]:
+        if isinstance(arg, (cudf.MultiIndex, pd.MultiIndex)):
+            if isinstance(arg, pd.MultiIndex):
+                arg = cudf.from_pandas(arg)
```

Comment:
Does this look like necessary branching?
```suggestion
        if isinstance(arg, pd.MultiIndex):
                arg = cudf.from_pandas(arg)
```

Reply:
oops - nice catch!